lib/matplotlib/backend_bases.py

```diff
@@ -293,13 +293,14 @@ def draw_path_collection(self, gc, master_transform, paths, all_transforms,
         """
         path_ids = []
         for path, transform in self._iter_collection_raw_paths(
-                master_transform, paths, all_transforms):
+                master_transform, paths, all_transforms, offsets):
```

Comment:
In this case the offsets are independent of the transforms (the latter used primarily for marker size).  Way down underneath they are combined for the Agg backend.  The vector backends can't however, because those formats would scale the stroke width as a result.

But maybe I don't understand your question/concern?